Documentation/CRDs/Cluster/network-providers.md

```diff
@@ -155,7 +155,7 @@ understand and implement these requirements.
     need to be an order of magnitude larger (or more) than the host address space to allow the
     storage cluster to grow in the future.
 
-If these prerequisites are not achievable, the remaining option is to set the Rook operator config
+If these prerequisites are not achievable, plan to set the Rook operator config
```

Comment:
even more direct...
```suggestion
If these prerequisites are not achievable, set the Rook operator config
```

Reply:
We don't actually want the users to set the value immediately, so I used "plan to set" instead of just "set". The setting is handled later.